contribute/design/proposal-cstor-orchestration.md

```diff
@@ -0,0 +1,36 @@
+# Managing cStor based OpenEBS Volumes. 
+
+## Background 
+
+
+## Design
+
+Provisioning of cStor based Volume involve two steps. 
+
+1. Creating cStor StoragePools:
+
+- Admin will create SPC that contains information like:
+  - type=block
+  - disks list (type of PDC or specific disks)
+  - nodes list (optional)
+
+- maya-cstor-operator (embedded into maya-apiserver), will be watching for SPCs (type=block). If nodes were not specified in the PVC, the maya-cstor-operator will query the kube-ectd for the availability of the disks and determine the nodes. 
```

Comment:
what if the same disk is available in multiple nodes?

Reply:
by same disk, do you mean an External Disk -- that is mounted on multiple nodes?